share/arno-iptables-firewall/environment

```diff
@@ -381,15 +405,23 @@ ip4tables_restore()
 {
   local result retval IFS=' '
 
-  result=`$IP4TABLES_RESTORE "$@" 2>&1`
+  exec 3>&1
+  err_result=`$IP4TABLES_RESTORE "$@" 2>&1 1>&3`
   retval=$?
 
```

Comment:
Similar fixes apply to `ip4tables_restore()` and `ip6tables_restore()`

Reply:
Well the logic there was always a little different since stdout there gets redirected by the caller. I'm not sure whether we should change this. Currently when an error occurs all output (stdout+stderr) gets sent to stderr so the user knows what's going on. What we could do is in case of an error also send stdout to stderr, so the caller still gets stdout to redirect. I hope your understand what I mean.